command/ssh.go

```diff
@@ -45,6 +47,15 @@ func (c *SSHCommand) Run(args []string) int {
 	if err := flags.Parse(args); err != nil {
 		return 1
 	}
+
+	if os.Getenv("VAULT_STRICT_HOST_KEY_CHECKING") != "" {
```

Comment:
Command line flags override env vars in Vault, but this has that logic flipped. Rather than set a default for the two parameters in the flags StringVars, check for the env var here, and if not set, and if the flag is empty, *then* set to the default.